src/content/en/updates/2020/11/puppetaria.md

```diff
@@ -0,0 +1,175 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Puppetaria: accessibility-first Puppeteer scripts. The Chrome DevTools engineering blog - by the developers who build the DevTools.
+
+{# wf_updated_on: 2020-11-10 #}
+{# wf_published_on: 2020-11-10 #}
+{# wf_tags: devtools-blog #}
+{# wf_featured_image: /web/updates/images/2020/11/puppetaria/puppetaria.png #}
+{# wf_featured_snippet: Puppetaria: accessibility-first Puppeteer scripts. #}
+{# wf_blink_components: N/A #}
+
+# Puppetaria: accessibility-first Puppeteer scripts {: .page-title }
+
+{% include "web/_shared/contributors/jobay.html" %}
+
+
+## Puppeteer and its approach to selectors {: .intro }
+
+Puppeteer is a browser automation library for Node: it lets you control a browser using a simple and modern JavaScript API.
+
+The most prominent browser task is, of course, browsing web pages. Automating this task essentially amounts to automating interactions with the webpage. 
+
+In Puppeteer, this is achieved by querying for DOM elements using string-based selectors and performing actions such as clicking or typing text on the elements. For example, a script that opens opens [developer.google.com], finds the search box, and searches for `puppetaria` could look like this:
```

Comment:
The `[developer.google.com]` link doesn't seem to work in the preview

Reply:
fixed